src/dbus_fast/message_bus.py

```diff
@@ -763,7 +763,7 @@ def __exit__(
                 exc_value: Optional[Exception],
                 tb: Optional[TracebackType],
             ) -> None:
```

Comment:
Looks like the typing needs to be adjusted as well

Reply:
fixed

Reply:
I've been slowing adding the missing typing when I have cycles. Its about 70% done and then I should be able to turn on mypy in the CI